src/vendor/wordpress/_wordpress.scss

```diff
@@ -111,6 +106,47 @@ $wp-button-gap: sizes.$list-inline-gap;
   }
 }
 
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our staggerItems
+ *    utility, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  /**
+   * 1. Setting a width of `100%` prevents resized and centered images from
```

Comment:
Nit: Comment numbering isn't necessary when there is only one line in the block.